workspaces/marketplace/plugins/marketplace/src/components/MarketplacePluginContent.tsx

```diff
@@ -255,7 +275,7 @@ const PluginPackageTable = ({ plugin }: { plugin: MarketplacePlugin }) => {
 export const MarketplacePluginContent = ({
   plugin,
 }: {
-  plugin: MarketplacePlugin;
+  plugin: MarketplacePlugin | MarketplacePackage;
```

Comment:
Naaa? Why is this needed? We had a MarketplacePackageContent before? I expect we lose a lot of type save-tyness if we say "this can be a package as well"